CMakeLists.txt

```diff
@@ -438,39 +463,19 @@ endif(WIN32)
 
 
 if (CLR_CMAKE_PLATFORM_UNIX)
-  add_definitions(-DPLATFORM_UNIX=1)
   add_definitions(-DFEATURE_PAL_SXS)
   add_definitions(-DFEATURE_COREFX_GLOBALIZATION)
+  add_definitions(-DFEATURE_PAL)
 
   if(IS_64BIT_BUILD)
     add_definitions(-DBIT64=1)
   endif(IS_64BIT_BUILD)
-
-  add_definitions(-DFEATURE_PAL)
-    
-  if(CLR_CMAKE_PLATFORM_LINUX)
-    add_definitions(-D__LINUX__=1)
-    if(CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64)
-      message("Detected Linux x86_64")
-      add_definitions(-DLINUX64)
-    elseif(CLR_CMAKE_PLATFORM_ARCH_ARM)
-      message("Detected Linux ARM")
-      add_definitions(-DLINUX32)
-    elseif(CLR_CMAKE_PLATFORM_ARCH_ARM64)
-      message("Detected Linux ARM64")
-      add_definitions(-DLINUX64)
-    else()
-      clr_unknown_arch()
-    endif()
-      
-  endif(CLR_CMAKE_PLATFORM_LINUX)
+  
   if(CLR_CMAKE_PLATFORM_DARWIN)
     message("Detected OSX x86_64")
```

Comment:
A nit: I would prefer the message to be up together with the other messages printing the detected platform, I would leave just the add_definitions here.